lib/src/main/java/io/ably/lib/transport/ConnectionManager.java

```diff
@@ -476,16 +482,11 @@ private StateIndication checkSuspend(StateIndication stateChange) {
 
 		/* FIXME: we might want to limit this behaviour to only a specific
 		 * set of error codes */
-		if(pendingConnect != null && !pendingConnect.fallback && checkConnectivity()) {
-			String[] fallbackHosts = Defaults.getFallbackHosts(options);
-			if(fallbackHosts != null && fallbackHosts.length > 0) {
-				/* we will try a fallback host */
-				StateIndication fallbackConnectRequest = new StateIndication(ConnectionState.connecting, null);
-				fallbackConnectRequest.useFallbackHost = true;
-				requestState(fallbackConnectRequest);
-				/* returning null ensures we stay in the connecting state */
-				return null;
-			}
+		if(pendingConnect != null && checkConnectivity()) {
```

Comment:
Ignore me, I did add an issue :confused: and didn't update this comment thread, see https://github.com/ably/ably-java/issues/93